src/plugins/workspace/server/plugin.ts

```diff
@@ -157,7 +158,13 @@ export class WorkspacePlugin implements Plugin<WorkspacePluginSetup, WorkspacePl
         enabled: true,
         permissionEnabled: isPermissionControlEnabled,
       },
+      dashboards: { isDashboardAdmin: true },
```

Comment:
I think the default value is better to set to `false`, then it's updated with `registerSwitcher`

Reply:
Updated